LEAF_Request_Portal/admin/templates/mod_groups.tpl

```diff
@@ -570,6 +586,16 @@ function getGroupList() {
                                                     });
                                                     dialog_confirm.show();
                                                 });
+
+                                                $('#reActivateMember_' + counter).on('click', function () {
+                                                    dialog_confirm.setContent('Are you sure you want to Reactivate this member?');
+                                                    dialog_confirm.setSaveHandler(function () {
+                                                        reactivateMember(groupID, res[i].userName);
+                                                        dialog_confirm.hide();
+                                                        dialog.hide();
+                                                    });
+                                                    dialog_confirm.show();
```

Comment:
This looks like what is calling the method above.  And it's decision loop is based solely on if the ajax call succeeds?

Reply:
I'm not sure I understand what you are asking.